lib/utils.js

```diff
@@ -22,7 +26,38 @@ function parseRoleId(role) {
   return matches && matches[1] || -1
 }
 
+/**
+ * Responds to a message with a prompt and returns the answer.
+ *
+ * @param {Message} msg Message to respond to
+ * @param {boolean} [sameAuthor] Whether the answer should be from the same author as the initial message
```

Comment:
Add default value for `sameAuthor` in the doc

Reply:
done